cliquet/resource/__init__.py

```diff
@@ -580,8 +580,17 @@ def postprocess(self, result, action=ACTIONS.READ):
         # when it's added
         if action != ACTIONS.READ:
             # the collection has been changed
+            if not isinstance(result, list):
+                result = [result]
+            impacted = {}
+            if action == ACTIONS.CREATE:
+                impacted = [{'new': r} for r in result]
+            elif action == ACTIONS.DELETE:
+                impacted = [{'old': r} for r in result]
+            elif action == ACTIONS.UPDATE:
+                impacted = [{'new': r, 'old': old} for r in result]
```

Comment:
Yes, I will document this. This will have to be rewritten when the notifications will be plugged with transactions. So far, Remy is right, only record can be modified at a time. But I wanted the event attribute to be unchanged when we go further with transactions